handlers/analytics/user_end.go

```diff
@@ -0,0 +1,73 @@
+package analytics
+
+import (
+	"context"
+	"database/sql"
+	"strings"
+	"time"
+
+	"github.com/lib/pq"
+	"github.com/livepeer/catalyst-api/handlers/misttriggers"
+)
+
+const USER_END_TABLE_NAME = "user_end"
+
+type AnalyticsHandler struct {
+	db *sql.DB
+}
+
+func NewAnalyticsHandler(db *sql.DB) AnalyticsHandler {
+	return AnalyticsHandler{db: db}
+}
+
+func (a *AnalyticsHandler) HandleUserEnd(ctx context.Context, payload *misttriggers.UserEndPayload) error {
+	// If there's nowhere to write to, this handler is a no-op
+	if a.db == nil {
+		return nil
+	}
+
+	streamNames := strings.Split(payload.StreamNames, ",")
```

Comment:
Probably this kind of string-parsing stuff should be handled in the trigger-parsing code, so then the `UserEndPayload` can just contain clean slices of data

Reply:
Yeah great call - in the initial implementation I hadn't realised we weren't just passing everything through as strings to the DB. I'll update now.